Core/ContentBlockerRulesSource.swift

```diff
@@ -41,11 +41,15 @@ class DefaultContentBlockerRulesSource: ContentBlockerRulesSource {
     }
 
     var tempListEtag: String? {
-        return UserDefaultsETagStorage().etag(for: .temporaryUnprotectedSites)
+        return UserDefaultsETagStorage().etag(for: .privacyConfiguration)
```

Comment:
Since we now have interface for that through `PrivacyConfigurationManager` (fetched/embedded fields contain pair config:etag) we should use that abstraction instead of reaching to storage directly.